clang/docs/ReleaseNotes.rst

```diff
@@ -991,6 +991,7 @@ Bug Fixes to C++ Support
 - Fixed assertions or false compiler diagnostics in the case of C++ modules for
   lambda functions or inline friend functions defined inside templates (#GH122493).
 - Clang now rejects declaring an alias template with the same name as its template parameter. (#GH123423)
+- Correctly determine the implicit constexprness of dependent lambdas. (#GH97958) (#GH114234)
```

Comment:
```suggestion
- Correctly determine the implicit constexprness of lambdas in dependent contexts. (#GH97958) (#GH114234)
```
Maybe this is a bit clearer because strictly speaking this is about cases where the lambda itself *isn’t* dependent.